### PR TITLE
Dekadal training

### PR DESCRIPTION
**Pull request by gabrieltseng**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -167,6 +176,11 @@
     model_kwargs = json.load(Path(config_dir / "default.json").open("r"))
     model = Presto.load_pretrained()
     best_model_path: Optional[Path] = default_model_path
+    if dekadal:
+        model = Presto.reinitialize_pos_embedding(model, max_sequence_length=72)
+        dekadal_model_path = default_model_path.parent / "dekadal_model.pt"
```
Comment (gabrieltseng) [inline train.py:181]: Saved as `dekadal_model.pt`

File: train.py [hunk 2]
```diff
@@ -332,14 +346,18 @@
...
-if finetuned_model is not None:
-    model_path = model_logging_dir / Path("models")
-    model_path.mkdir(exist_ok=True, parents=True)
-    finetuned_model_path = model_path / "finetuned_model.pt"
-    torch.save(model.state_dict(), finetuned_model_path)
+
+#if finetuned_model is not None:
+model_path = model_logging_dir / Path("models")
+model_path.mkdir(exist_ok=True, parents=True)
+finetuned_model_path = model_path / "finetuned_model.pt"
```
Comment (gabrieltseng) [inline train.py:358]: https://github.com/WorldCereal/presto-worldcereal/pull/35/files#r1547829997 

but here `finetuned_model.pt` is loaded

Comment (giollimirgia) [reply train.py:358]: that's where the `finetuned_model.pt` is saved after finetuning the `dekadal_model.pt` (which is being loaded here https://github.com/WorldCereal/presto-worldcereal/blob/dataset-10days/train.py#L340 )

**Comment by gabrieltseng** [comment]
Superseded by #75 , closing

**Pull request closed without merging**